File: informatics.mccme.ru/Python/Chapter 4/O/O.py
a = int(input())
b = int(input())
c = int(input())
d = int(input())

for i in range(0, 1001):
    if (a * i ** 3 + b * i ** 2 + c * i + d == 0):
        print(i, end = ' ')
         

File: informatics.mccme.ru/Input-output/O/O.c
#include <stdio.h>

int main() {
    freopen("input.txt", "r", stdin);
    freopen("output.txt", "w", stdout);
    
    int n, a, b;
    scanf("%d%d%d", &a, &b, &n);
    
    a *= n;
    b *= n;
    
    a += b / 100;
    b %= 100;
    
    printf("%d %d", a, b);
    
    return 0;
}
